src/firefly/js/ui/dynamic/DynComponents.jsx

```diff
@@ -177,6 +177,7 @@ export function PositionAndPolyFieldEmbed({fieldDefAry, plotId, toolbarHelpId, i
                                        otherComponents, WrapperComponent}) {
 
     const polyType = findFieldDefType(fieldDefAry, POLYGON);
+    //const uploadType = findFieldDefType(fieldDefAry, UPLOAD);
     const posType = findFieldDefType(fieldDefAry, POINT) ?? findFieldDefType(fieldDefAry, POSITION);
```

Comment:
@robyww following up on my slack question, this is what I meant. I don't think we'll get any upload info from the `fieldDefAry`, right? so is it safe to just set `useUpload = true` (see line 204) before the call to `EmbeddedPositionSearchPanel`?  that'll always give an option to upload a table in all of DCE...